src/Analyzer/ClassDescriptionBuilder.php

```diff
@@ -8,47 +8,40 @@
 class ClassDescriptionBuilder
 {
     /** @var list<ClassDependency> */
-    private $classDependencies = [];
+    private array $classDependencies = [];
 
-    /** @var ?FullyQualifiedClassName */
-    private $FQCN;
+    private ?FullyQualifiedClassName $FQCN = null;
 
     /** @var list<FullyQualifiedClassName> */
-    private $interfaces = [];
+    private array $interfaces = [];
 
-    /** @var ?FullyQualifiedClassName */
-    private $extend;
+    /** @var list<FullyQualifiedClassName> */
+    private array $extend = [];
```

Comment:
Is it better to use the plural "extends" like for other properties and in other files?

Reply:
Yes, it makes sense 👍🏻